### PR TITLE
tests, Use RandName for creating random VMI names

### PR DESCRIPTION
**Pull request by oshoval**
Following the work done at
https://github.com/kubevirt/kubevirt/pull/4605 https://github.com/kubevirt/kubevirt/pull/4609
which updated tests to use a shorter random VMI suffix, in order to ease logs readability,

Use `libvmi.RandName(baseName)` in tests which create a random VMI name,
in order to unify the interface and the VMI name pattern in the logs
for those tests.

**Special notes for your reviewer**:
* On a follow PR, it might worth to check if the tests which use a fixed VMI name `"testvmi"`,
are running in a `[Serial]` mode due to this fact, or might collide in case they run in a parallel mode,
and adapt them (in one way or another).

**Release notes**:
```release-note
None
```

### CODE REVIEW

**Comment by oshoval** [comment]
/hold
until #4609 is merged
/cc @dankenigsberg

File: tests/libvmi/vmi.go
Comment (dankenigsberg) [inline tests/libvmi/vmi.go:47]: I don't see the value of this function. We are in the vmi.go module, so the "Vmi" prefix is redudent.

Comment (oshoval) [reply tests/libvmi/vmi.go]: Updated

File: tests/libvmi/vmi.go
Comment (EdDev) [inline tests/libvmi/vmi.go:47]: I like less the fact that this function no longer accepts a prefix string, in tests with more than one VMI it may be more useful to allow it in the future.
I guess one way to overcome it is to use a variable length arguments to implement a "default" but that is ugly by itself.
I would prefer you use the default constant name in the callers at this stage.

**Review by EdDev — CHANGES_REQUESTED** [review]
Commit messages:
If you want to ref a prev change, please do it at the tail of the commit message and not at the start.
Something like "Consistently use the same VMI name pattern in all tests to ease debugging...".

I also do not think it merits two commits (actually 4 if I consider the prev changes).

**Comment by EdDev** [comment]
> I also do not think it merits two commits (actually 4 if I consider the prev changes).

Ahh, I missed that the 1st commit is a rebase and is not part of this PR.

**Comment by oshoval** [comment]
Addressed comments

**Review by EdDev — APPROVED** [review]
Thanks.

**Comment by oshoval** [comment]
Rebased and aligned the commit message with the PR description.

/hold cancel

**Review by EdDev — APPROVED** [review]

**Pull request merged**
